templates/tfengine/components/cicd/triggers/apply/main.tf

```diff
@@ -11,6 +11,11 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */ -}}
+locals {
+  terraform_root = var.terraform_root == "/" ? "." : var.terraform_root
+  terraform_root_prefix = var.terraform_root == "/" ? "" : "${var.terraform_root}/"
```

Comment:
I'm looking at https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/templates/tfengine/components/cicd/triggers.tf#L21 and i think there is subtle differences there. You probably should use local.terraform_root here so the case where `var.terraform_root` (or `local.terraform_root`) is `.` is considered. And in that case, `terraform_root_prefix` is set to `""`.

Reply:
You're right, I changed it for every trigger module.

Thanks for checking it out with that detail. I'm resolving